tensorflow/core/kernels/mkl/mkl_fused_batch_norm_op_test.cc

```diff
@@ -159,7 +159,7 @@ class CommonTestUtilities : public OpsTestBase {
     ASSERT_EQ(offset_backprop.shape(), mkl_offset_backprop.shape());
 
     test::ExpectClose(output, mkl_output, 1e-5);
-    test::ExpectClose(scale_backprop, mkl_scale_backprop, 1e-5);
+    test::ExpectClose(scale_backprop, mkl_scale_backprop, 1e-5, /*rtol*/ 1e-5);
```

Comment:
Nit: For completeness.
```suggestion
    test::ExpectClose(scale_backprop, mkl_scale_backprop, /*atol=*/1e-5, /*rtol=*/1e-5);
```

Reply:
I have addressed the comment. Thank you!